src/JLFzf.jl

```diff
@@ -42,21 +42,19 @@ return selected string.
 Additional arguments `args` for `fzf` are allowed.
 """
 function inter_fzf(in_str::String, args...)
-    fzf_jll.fzf() do exe
-        if length(args) == 0
-            return read(
-                pipeline(Cmd(`$exe`, ignorestatus = true), stdin = IOBuffer(in_str)),
-                String,
-            ) |> chomp
-        else
-            return read(
-                pipeline(
-                    Cmd(`$exe $(args)`, ignorestatus = true),
-                    stdin = IOBuffer(in_str),
-                ),
-                String,
-            ) |> chomp
-        end
+    if length(args) == 0
+        return read(
+            pipeline(Cmd(`$(fzf_jll.fzf())`, ignorestatus = true), stdin = IOBuffer(in_str)),
```

Comment:
```suggestion
            pipeline(Cmd(fzf_jll.fzf(), ignorestatus = true), stdin = IOBuffer(in_str)),
```
but I'm confused by why there's an if, could always use the form in the `else` branch, no?

Reply:
The only thing I changed the use of the jll, I barely looked at it otherwise, so I don't know why anything is as it is.  I will happily commit any suggestions and am completely agnostic on all this.